aerial_robot_simulation/launch/simulation.launch

```diff
@@ -9,6 +9,9 @@
   <arg name="spawn_y" default="0.0"/>
   <arg name="spawn_z" default="0.0"/>
   <arg name="spawn_yaw" default="0.0"/>
+  <arg name="use_model_with_sim_flag" default="false"/>
```

Comment:
I do not think that using this flag is the best solution. We can exclude the joint named `torsion${self}` in our robot model loading phase. Then we do not need two urdf model (`robot_description` and `robot_description_sim`)